### PR TITLE
Add objdump flag to print section offsets instead of file offsets

### PR DESCRIPTION
**Pull request by dschuff**
This is useful for working with debug info, which uses section offsets

### CODE REVIEW

File: src/binary-reader-objdump.cc [hunk 1]
```diff
@@ -195,6 +196,12 @@ void BinaryReaderObjdumpBase::PrintRelocation(const Reloc& reloc,
   printf("\n");
 }
 
+Offset BinaryReaderObjdumpBase::GetPrintOffset(Offset offset) const {
+  return
+      options_->section_offsets ?
+      offset - GetSectionStart(BinarySection::Code) : offset;
+}
```
Comment (sbc100) [inline src/binary-reader-objdump.cc:203]: I find the multi-line ternery expression s less easy to read.  How about:

```
if (options_->section_offsets) {
   return offset - GetSectionStart(BinarySection::Code);
}
return offset;
```

**Review by sbc100 — APPROVED** [review]

**Comment by dschuff** [comment]
There could maybe be some more places where it might be useful to also show section offsets under this flag, but this is useful as-is too.

File: src/binary-reader-objdump.cc (hunk 1 above)
Comment (dschuff) [reply src/binary-reader-objdump.cc:203]: oops, I meant to let clang-format fix this before uploading. here's what it comes up with:
```
  return options_->section_offsets
             ? offset - GetSectionStart(BinarySection::Code)
             : offset;
```

Comment (dschuff) [reply src/binary-reader-objdump.cc:203]: That looks better than the 2-line version, but I can do the if/else version if you prefer too

Comment (sbc100) [reply src/binary-reader-objdump.cc:203]: I guess that is nicer yes.   lgtm

**Pull request merged**